DPKI_operations.md

```diff
@@ -0,0 +1,38 @@
+# Chapter on PKI/DPKI operations
+
+
+## Introduction to PKI operations:
+In the public key infrastructure (PKI), there are two main operations, which are: **signing request**, and **revocation request**. 
```

Comment:
Do we have any references for this?

Reply:
no, I don't have them. <br> A couple of days ago I found something on the DigiCert websites. Should I add the site on the references sections?